prussia_rt/src/lib.rs

```diff
@@ -41,7 +61,20 @@ pub unsafe extern "C" fn _rust_start() -> ! {
         fn main() -> !;
     }
 
+    println_ee!("rt - Hello world!");
+
     zero_bss();
 
+    println_ee!("rt - BSS zero-ed out.");
+
+    initialise_exception_vectors();
```

Comment:
I think it might be smarter to initialise the vectors before zeroing bss in case the elf is somehow corrupt and bss points to invalid address space.

Reply:
Fixed.